### DISCUSSION TITLE
Preview 13 + VS 2022 17.2P1 intellisense bugging out?

### DISCUSSION THREAD

**Discussion by danielftz** [dotnet/maui#4785] [Q&A] [answered]
I have updated to preview13, and updated vs2022 to latest 17.2p1, and I am currently developing cross platform mobile apps (IOS and android). I am experiencing an issue where if I navigate to any code under Platforms.iOS folder while having the solution target android, IntelliSense is giving me error for any IOS specific packages and classes like UIKit, CoreGraphics, etc. (And vice versa). Is anyone having this issue? And is there any workaround for this?

**Comment by JohnHDev** [comment, marked as answer]
@danielftz yes and I reported it as a bug, but there is a simple workaround:
https://github.com/dotnet/maui/issues/4726
